src/pages/ListScholarships.jsx

```diff
@@ -56,36 +58,33 @@ function ListScholarships() {
       </Helmet>
       <Drawer
         sx={{
-          display: { xs: 'none', md: 'block' },
           flexShrink: 0,
           width: drawerWidth,
           '& .MuiDrawer-paper': {
             width: drawerWidth,
             position: 'absolute',
             boxSizing: 'border-box',
           },
-          position: 'sticky',
+          position: { xs: 'static', md: 'sticky' },
           overflowY: 'auto',
         }}
-        variant="permanent"
+        open={drawerOpen || isDesktop}
+        variant={isDesktop ? 'permanent' : 'temporary'}
         anchor="left">
-        <FilterPanel />
+        <FilterPanel closePanel={() => setDrawerOpen(false)} />
       </Drawer>
+
       <Container
         maxWidth="md"
         component="main"
         ref={ref}
         sx={{
           bgcolor: 'background.default',
           flexGrow: 1,
-          padding: 2,
           position: 'sticky',
           overflowY: { md: 'auto' },
         }}>
-        <Typography variant="h4" component="h1" align="center" style={{ p: 1 }}>
-          {t('general.scholarships')}
-        </Typography>
-        <FilterBar />
+        <FilterBar openFilter={() => setDrawerOpen(true)} />
```

Comment:
The surrounding container introduces margins to this. How about sticking it outside of the container? Though to make it scrollable with the rest of the main content you'll need to surround both this and the container inside a box that has `position: 'sticky'` and `overflowY` set instead of the container here.

Reply:
Also to keep the filter bar contents aligned with the container it'll need to inside of a container with the same `maxWidth` itself. So in summary
```
Scrollable Box
  - FilterBar
    - Container md
      - buttons
  - Container md
    - ScholarshipList
```